src/components/admin/hotels/save-hotel/manage-attachments.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+    Box,
+    Button,
+    FormControl,
+    Grid,
+    List,
+    ListItem,
+    ListItemIcon,
+    ListItemSecondaryAction,
+    ListItemText,
+    Paper
+} from '@material-ui/core';
+import InsertDriveFileIcon from '@material-ui/icons/InsertDriveFileOutlined';
+import AttachmentIcon from '@material-ui/icons/Attachment';
+
+import { Link } from 'react-router-dom';
+
+import './manage-attachments.css';
+
+function ManageAttachments({ attachments, onFileAdd, onAttachmentRemove }) {
+
+    function handleFileChange(e) {
+        const files = e.target.files;
+        if (files.length) {
+            onFileAdd(files[0]);
+        }
+    }
+
+    return (
+        <Grid container direction="column" justify="center" alignItems="stretch">
+            <Grid item>
+                <Box mb={1}>
+                    <FormControl fullWidth>
+                        <input
+                            id="contained-button-file"
+                            className="contained-button-file-input"
+                            type="file"
+                            hidden
+                            onChange={handleFileChange}
+                        />
+                        <label htmlFor="contained-button-file">
+                            <Button variant="contained" size="small" color="default" component="span" fullWidth startIcon={<InsertDriveFileIcon />}>
+                                Browse and add files
+                                </Button>
+                        </label>
+                    </FormControl>
+                </Box>
+            </Grid>
+            <Grid item>
+                <Paper className="list-paper" variant="outlined">
```

Comment:
Instead of always showing an area for selected files, show `<Paper>` if one or more files are added. Height will be automatically figured out. When a lot of files will be selected Dialogs height will grow, and after that Dialogs scroll will appear

Reply:
Fixed.